### ISSUE TITLE
UZF budget file (binary output) not being successfully read

### ISSUE THREAD

**Issue by emorway-usgs** [modflowpy/flopy#872]
Using the attached MF6 model (~20 sec runtime, 2 stress periods), I try to load the UZF binary output file with:

```
import flopy.utils.binaryfile as bf
uzf_bud_file = 'gwf_modsim.uzf.bud'
uzfobj = bf.CellBudgetFile(uzf_bud_file)
```

But the cmd line has been spinning on that last command for nearly 20 minutes, and the total file size of the uzf binary budget file is 5Mb.  Not sure if the issue is with flopy, or with MF6's writing of the UZF budget file?  I have not included the mf6 executable in the attached, one will need to retrieve a copy of the latest develop version to run the attached model.  
[uzf_budget_file_bug_report.zip](https://github.com/modflowpy/flopy/files/4601403/uzf_budget_file_bug_report.zip)

**Comment by emorway-usgs** [comment]
@jlarsen-usgs This seemed to fix my problem:

`uzfobj = bf.CellBudgetFile(uzf_bud_file, precision='double')`

However, @langevin-usgs suggested that one of your recent fixes (#823) should have trapped for this.  Thus, it may be worth double-checking your fix with the model attached to this issue?  I tried `...,precision='auto')`, but that bombed as well.

**Comment by jlarsen-usgs** [comment]
@emorway-usgs 

I definitely take a look at this. The CellBudgetFile code should trap for precision, but it's possible that I missed something when I pushed that update.
